CHANGELOG.md

```diff
@@ -12,6 +12,26 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Breaking changes
 
+### Deprecations
+
+### New features
+
+### Fixed bugs
+
+## [0.28.0] - 2025-04-02
+
+### Release highlights
+
+* All outstanding "pinned" issues in the GitHub repository have been resolved, including a long-standing bug relating to empty files in the built-in diff editor.
```

Comment:
(Optional): I'd either skip these first point, or link to some of the relevant bugs.

In the latter case, it might be sufficient to talk about the one specific bug you called out.

If we want to mention them all, perhaps we could say something like:

> Long-standing bugs relating to git remote compatibility (#XXXX, #YYYY) and empty files in the built-in git editor (#ZZZZ) were resolved.

That said, we could keep what you have if it's easier.

Reply:
@ilyagr I couldn't remember what all of the historical pinned files were, which is the main reason I didn't list them all 🤣. I changed this to just list that one bug by link and demoted it to a later point in the list.